src/libraries/System.ComponentModel.Annotations/src/Resources/Strings.resx

```diff
@@ -138,6 +147,15 @@
   <data name="MinLengthAttribute_ValidationError" xml:space="preserve">
     <value>The field {0} must be a string or array type with a minimum length of '{1}'.</value>
   </data>
+  <data name="LengthAttribute_InvalidMinLength" xml:space="preserve">
+    <value>LengthAttribute must have a MinimumLength value that is zero or greater.</value>
+  </data>
+  <data name="LengthAttribute_InvalidMaxLength" xml:space="preserve">
+    <value>LengthAttribute must have a MaximumLength value that is greater than or equal to MinimumLength.</value>
+  </data>
```

Comment:
nit: should we include the wrong min/max value in the message? 

LengthAttribute have a MinimumLength value {0} which is less than zero.

or whatever.

Reply:
This is following convention of MinLengthAttribute and MaxLengthAttribute when it comes to failing on invalid attribute bounds:

https://github.com/dotnet/runtime/blob/cb8d49aaa4aa4f76e4cee0c2228651e1093188e9/src/libraries/System.ComponentModel.Annotations/src/Resources/Strings.resx#L135-L137

Reply:
do we need to stick with thta?

Reply:
I would much rather see the error containing the allowed range and the value that was supplied. Folks see these errors as a result of having made an error in some JSON config file and providing the most meaningful errors possible helps a lot.

Reply:
This particular error mode concerns validating the constructor parameters on the attribute itself, not validating any runtime inputs. Actual validation errors are surfaced using the `LengthAttribute_ValidationError` resource defined below:

```xml
<value>The field {0} must be a string or collection type with a minimum length of '{1}' and maximum length of '{2}'.</value>
```